NuGet.Config

```diff
@@ -1,10 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why does this not have the changes from nimak/main? (asking because formatting is different)